### ISSUE TITLE
scalafmt align.tokens.add doesn't work with sbt's := operator

### ISSUE THREAD

**Issue by mdedetrich** [scalameta/scalafmt#2618]
This template is a guideline, not a strict requirement.
- **Version**: 2.7.5
- **Integration**: scalafmt cli built with graalvm (see https://aur.archlinux.org/packages/scalafmt-native-bin)
- **Configuration**:
```
version = 2.7.5
preset = default
align.preset = most
maxColumn = 120
project.git = false
align.tokens.add = [
  {code = ":=", owner = "(Enumerator.Val|Defn.(Va(l|r)|Def|Type))"}
]
align.openParenDefnSite = true
align.openParenCallSite = true
align.arrowEnumeratorGenerator = true
danglingParentheses.preset = true
rewrite.rules = [RedundantBraces, RedundantParens]
```

## Steps

Given code like this:
```scala
ThisBuild / scalaVersion := "2.13.6"
ThisBuild / version := "0.1.0-SNAPSHOT"
```

When I run scalafmt like this:

```bash
scalafmt
```

## Problem

Scalafmt formats code like this:
```scala
ThisBuild / scalaVersion := "2.13.6"
ThisBuild / version := "0.1.0-SNAPSHOT"
```

## Expectation

I would like the formatted output to look like this:
```scala
ThisBuild / scalaVersion := "2.13.6"
ThisBuild / version      := "0.1.0-SNAPSHOT"
```

In other words, I expect the `:=` to vertically align just like `=` does.

## Notes

See gitter conversation at https://gitter.im/scalameta/scalafmt?at=60dc7e9ab60c3137c39e6986 . 

I am not entirely sure if `scalafmt` adjusts its behaviour depending on the filename, but in this case it is being run against a project that is containing a `build.sbt` file.

Also note that the

```
align.tokens.add = [
  {code = ":=", owner = "(Enumerator.Val|Defn.(Va(l|r)|Def|Type))"}
]
````

configuration is taken from https://github.com/scalameta/scalafmt/blob/08c6798a40188ce69b3287e3026becfbd540a847/scalafmt-core/shared/src/main/scala/org/scalafmt/config/AlignToken.scala#L46 (since I am expecting it to work just like equals does).

**Comment by kitbellew** [comment]
> align.tokens.add = [
>   {code = ":=", owner = "(Enumerator.Val|Defn.(Va(l|r)|Def|Type))"}
> ]
> ...
> ```scala
> ThisBuild / scalaVersion := "2.13.6"
> ThisBuild / version := "0.1.0-SNAPSHOT"
> ```

But in this expression, `:=` is not owned any of those tree types. the documentation tells you how to figure out the owner: https://scalameta.org/scalafmt/docs/configuration.html#aligntokens

**Comment by mdedetrich** [comment]
Ah thanks, I figured out the issue. According to the AST explorer I should be using `Term.ApplyInfix` however when using  https://github.com/scalameta/scalafmt/blob/08c6798a40188ce69b3287e3026becfbd540a847/scalafmt-core/shared/src/main/scala/org/scalafmt/config/AlignToken.scala#L44 as a reference I just did

```
align.tokens.add = [
  {code = ":=", owner = "applyInfix"}
]
```

Which of course didn't work, doing

```
align.tokens.add = [
  {code = ":=", owner = "Term.ApplyInfix"}
]
```

Fixed the problem, closing the issue
